code/game/machinery/status_display.dm

```diff
@@ -163,6 +163,15 @@
 	if(maptext)
 		maptext = ""
 
+/obj/structure/machinery/status_display/proc/set_sec_level_picture()
+	switch(security_level)
+		if(SEC_LEVEL_GREEN)
+			set_picture("default")
+		if(SEC_LEVEL_BLUE)
+			set_picture("bluealert")
+		if(SEC_LEVEL_RED, SEC_LEVEL_DELTA)
+			set_picture("redalert")
```

Comment:
redundant with `sec_changed` above

Reply:
They're two different procs, functionally (one depends on an arg, one depends on security level). While it _could_ be done with just one proc, this seems more readable to me